src/elements/content-sidebar/additional-tabs/AdditionalTabs.js

```diff
@@ -19,6 +20,8 @@ type State = {
     isLoading: boolean,
 };
 
+const AdditionalTabWithFtux = withFtuxTooltip(AdditionalTab);
```

Comment:
Could also use the hoc directly in `AdditionalTab`, i.e. `export default withFtuxTooltip(AdditionalTab)`

Reply:
I agree with @ConradJChan. The details of how `AdditionalTab` work should remain hidden from its parent.